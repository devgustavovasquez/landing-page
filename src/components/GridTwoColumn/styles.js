import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';

export const Container = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: 1fr 2fr;
    align-items: center;
    gap: ${theme.spaces.large};

    @media ${theme.medias.lteMedium} {
      grid-template-columns: 1fr;
      text-align: center;
    }

    ${Title} {
      margin-bottom: ${theme.spaces.xlarge};
    }
  `}
`;

export const TextContainer = styled.div`
  ${({ theme }) => css`
    @media ${theme.medias.lteMedium} {
      margin-bottom: ${theme.spaces.large};
    }
  `}
`;

export const ImageContainer = styled.div`
  ${({ theme }) => css``}
`;

export const Image = styled.img`
  ${({ theme }) => css`
    width: 100%;
  `}
`;
