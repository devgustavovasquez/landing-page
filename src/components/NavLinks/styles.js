import styled, { css } from 'styled-components';

export const Container = styled.nav`
  ${({ theme }) => css`
    display: flex;
    flex-flow: row wrap;

    @media ${theme.medias.lteMedium} {
      flex-flow: column wrap;
      align-items: center;
    }
  `}
`;
