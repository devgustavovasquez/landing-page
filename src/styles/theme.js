export const theme = {
  colors: {
    mainBg: 'pink',
    secondaryBg: 'blue',
  },
  fonts: {},
  spaces: {},
};
