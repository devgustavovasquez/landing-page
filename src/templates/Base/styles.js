import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    padding-top: 5.4rem;

    @media ${theme.medias.lteMedium} {
      padding-top: 0;
    }
  `}
`;
