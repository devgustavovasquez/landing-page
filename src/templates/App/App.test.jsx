import { screen } from '@testing-library/react';
import Home from '.';
import { renderTheme } from '../../styles/render-theme';
import { theme } from '../../styles/theme';

test('renders learn react link', () => {
  renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', { name: 'hello' }).parentElement;
  expect(headingContainer).toHaveStyle({
    background: theme.colors.mainBg,
  });

  expect(headingContainer).toHaveStyleRule('background', 'pink');
  expect(headingContainer).toMatchSnapshot();
});
