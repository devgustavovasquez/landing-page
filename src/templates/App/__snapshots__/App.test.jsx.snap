// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  background: pink;
}

<div
  class="c0"
>
  <h1>
    hello
  </h1>
</div>
`;
